bibedit/record_proceedings.xml

```diff
@@ -35,13 +39,19 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
     <subfield code="m">VOLATILE:email</subfield>
     <subfield code="u">VOLATILE:Affiliation</subfield>
   </datafield>
+  <datafield tag="210" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:title_variation - extra words/phrasings for title </subfield>
+  </datafield>
   <datafield tag="242" ind1=" " ind2=" ">
     <subfield code="a">VOLATILE:translated title (English)</subfield>
     <subfield code="b">VOLATILE:translated subtitle (English)</subfield>
   </datafield>
   <datafield tag="245" ind1=" " ind2=" ">
-    <subfield code="a">VOLATILE:Original title</subfield>
-    <subfield code="b">VOLATILE:subtitle</subfield>
+    <subfield code="a">VOLATILE:Title of Proceedings if different from Conference itself</subfield>
+  </datafield>
+  <datafield tag="245" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:Proceedings, Title (acronym)</subfield>
+    <subfield code="b">VOLATILE:City, Country, Date (e.g. September 24–28, 2013)</subfield>

```

Comment:
That's a very good question. Actually, I'm not a big fan of that line, but that's what we do at the moment. 
There is some ongoing discussion on the subtitle but no final decision so far. 

-- 
Florian Schwennsen 
Deutsches Elektronen-Synchrotron DESY 
Building 01 Room O1.446 
phone: +49-40-8998-6190 

> From: "Samuele Kaplun" <notifications@github.com>
> To: "inspirehep/inspire" <inspire@noreply.github.com>
> Cc: "Florian Schwennsen" <florian.schwennsen@desy.de>
> Sent: Monday, 8 June, 2015 13:36:28
> Subject: Re: [inspire] bibedit: update record templates (#88)

> In bibedit/record_proceedings.xml :
> >    <datafield tag="242" ind1=" " ind2=" ">
> >      <subfield code="a">VOLATILE:translated title (English)</subfield>
> >      <subfield code="b">VOLATILE:translated subtitle (English)</subfield>
> >    </datafield>
> >    <datafield tag="245" ind1=" " ind2=" ">
> > -    <subfield code="a">VOLATILE:Original title</subfield>
> > -    <subfield code="b">VOLATILE:subtitle</subfield>
>> +    <subfield code="a">VOLATILE:Title of Proceedings if different from
> > Conference itself</subfield>
> > +  </datafield>
> > +  <datafield tag="245" ind1=" " ind2=" ">
> > +    <subfield code="a">VOLATILE:Proceedings, Title (acronym)</subfield>
>> +    <subfield code="b">VOLATILE:City, Country, Date (e.g. September 24–28,
> > 2013)</subfield>

> Silly question, why are we putting this in 245__b rather than i 260/269?

> —
> Reply to this email directly or view it on GitHub .